src/BlazorWebView/samples/BlazorWinFormsApp/Form1.cs

```diff
@@ -17,25 +17,17 @@ public partial class Form1 : Form
 		public Form1()
 		{
 			var services1 = new ServiceCollection();
-			services1.AddWindowsFormsBlazorWebView(new()
-			{
+			services1.AddWindowsFormsBlazorWebView();
```

Comment:
Do we still need the platform-specific registration APIs? Looking at the implementation it seems like it would be enough to just have AddBlazorWebView.

Reply:
They still need to register the element that holds the settings in DI

Reply:
It's up to us to decide what public API we want. If we wanted it to be just `AddBlazorWebView` then I think we could do so (absence of the service registration would imply defaults).

By all means we can have platform-specific APIs if we want to, but it doesn't seem to be strictly required for this specific functionality.

Reply:
> It's up to us to decide what public API we want. If we wanted it to be just `AddBlazorWebView` then I think we could do so (absence of the service registration would imply defaults).

This is not a pattern that we apply elsewhere in ASP.NET Core (in general) as it breaks things like constructor injection.

Reply:
Added the marker